provider/resources.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/pulumi/pulumi-xyz/provider/pkg/version"
 	"github.com/pulumi/pulumi/sdk/v3/go/common/resource"
 	"github.com/terraform-providers/terraform-provider-xyz/xyz"
+	// uncomment if using experimental token mapping (below)
+	// "github.com/pulumi/pulumi/sdk/v3/go/common/util/contract"
```

Comment:
We don't need this anymore (and they are not experimental). Instead we need `"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"`.

Reply:
My comment was wrong for the import path. Sorry.

Reply:
I should have caught that, thanks